src/constants.ts

```diff
@@ -85,4 +85,12 @@ export const mappingSortProps: MappingSortProps = {
   activeBids: {
     activeBids: -1,
```

Comment:
What is this sort option for? I think we don't need this option.

Reply:
I added it because History has the same option, I agree it probably doesnt do much but I wanted it to be consistent with History

Reply:
@deep-quality-dev It was a requirement to be able to sort bids by time as well as price.  It's been a minute but I believe the plan is to use this to see latest bids (and their associated glance) in a a feed type view.

Reply:
Make sense, thanks guys

Reply:
Just want to confirm @Will-Hyde this will sort domains by their most recent bid? 

E.g.
     Domain A has 1 active bid at 2pm
     Domain B has 1 active bid at 3pm 
     Domain C has 2 active bids at 1pm and 4pm.

     Domain A has another active bid placed 5pm
     
     I would expect the descending (most recent) sort order to be: A, C, B

Reply:
Yes: if you sort by  "activeBids.time" descending, it will return all results based on active bids from new to old. 

It can be a bit confusing because every domain can have many active bids, but for this example, it will use the most recent bid they have in their activeBid array